### PR TITLE
Add circle ci config

### PR DESCRIPTION
**Pull request by dereklieu**
Adds circle ci configuration. I prefer circle as I've found it much faster than Travis.

### CODE REVIEW

**Comment by dereklieu** [comment]
Note, I've just set the default branch to `development` since that's where the majority of pull requests should be made against.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: developmentseed/circleci:node8.4-python2.7
```
Comment (kriswep) [inline .circleci/config.yml]: Any specific reason you choosed this image over the 'official' circleci node image (https://hub.docker.com/r/circleci/node/)? Is that for python?
I think using image 
`- image: circleci/node:8` or `- image: circleci/node:8.9` would be a good call.
Please note that v8.4 is  actually not LTS (8.9.0 is).

**Review by kriswep — CHANGES_REQUESTED** [review]
Using circleci would work great imo. 
We could also consider adding a badge...

I was just wondering over the choosen node image, please see my line comment.

**Comment by dereklieu** [comment]
Yeah good point, I changed it to `circleci/node:8.9`

**Pull request merged**